README.md

```diff
@@ -69,10 +69,7 @@ This project is licensed under the Apache-2.0 license.
 
 ### Retrieving user variables
 
-After installation, all user-specific variables (such as `USER_POOL_APP_CLIENT_ID`) can be found in the `Info_Output.log` file. You can also retrieve these values by running the following command:
-```
-serverless info --verbose --region <REGION> --stage <STAGE>. 
-```
+After installation, all user-specific variables (such as `USER_POOL_APP_CLIENT_ID`) can be found in the `Info_Output.log` file.
```

Comment:
Sadly no, I couldn't find a way other than using `yarn deploy` and appending the output to INFO_OUTPUT.log.

Reply:
Gotcha. We can add additional information about how to get it from the console if that's the case?

Reply:
There wasn't a way I could find for cdk to get the information after deployment through the console, hence why I've output all the information to INFO_OUTPUT.log so that it isn't lost after.